pkgs/top-level/all-packages.nix

```diff
@@ -9344,11 +9344,9 @@ with pkgs;
     }
   );
 
-  readline = readline82;
-
   readline70 = callPackage ../development/libraries/readline/7.0.nix { };
 
-  readline82 = callPackage ../development/libraries/readline/8.2.nix { };
```

Comment:
You should leave a deprecation message if you're going to remove the versioned package attr.

```nix
   readline82 = throw "readline 8.2 has been removed. Use `readline` instead"; # added 2025-07-11
```

Reply:
IMHO it's obvious with versioned attrs that the specific version was removed. I've never added one in the past. This is only useful when whole packages are removed.

Is this written down somewhere? We should change that in that case.

Reply:
Hm, I just applied the general rule that any deleted package should have an alias for it that explains the deprecation (and/or gives a replacement). But indeed in this case it's very clear as-is...